UITests/SecurityTests.swift

```diff
@@ -129,24 +129,6 @@ class SecurityTests: KIFTestCase {
           EarlGrey.selectElement(with: grey_allOf(closeButtonMatchers)).perform(grey_tap())
       }
 
-    func testDataURL() {
```

Comment:
This was removed in a previous commit, I guess you need to rebase main to have latest changes correctly